src/components/common/ProgressIndicatingImage.js

```diff
@@ -23,10 +23,20 @@ class ProgressIndicatingImage extends Component {
         super(props)
         this.state = {
             imageIsLoaded: false,
-            imageOpacity: new Animated.Value(0)
+            imageOpacity: new Animated.Value(0),
+            viewDimensions: {
+                width: 0,
+                height: 0
+            },
+            imageDimensions: {
+                width: 0,
+                height: 0
+            }
         }
         this.animateImageIfLoaded = this.animateImageIfLoaded.bind(this)
         this.unlinkImageOnLoad = false
+
+        this.animateImageIfLoaded()
```

Comment:
Is there a reason this is done here instead of within a lifecycle method?

Reply:
No, this should probably go in `componentDidMount`